main.js

```diff
@@ -21,4 +177,115 @@ navLink.addEventListener('click', () => {
     document.body.style.overflowY = 'scroll';
     menu.classList.toggle('show-menu');
   });
+
+ 
+
+  const cardWorks=document.querySelector('.card-works');
+  const topDesktop=document.createElement('div');
+  topDesktop.classList.add('top-desktop');
+  cardWorks.appendChild(topDesktop);
+
+  const headingPart=document.createElement('div');
+  headingPart.classList.add('heading-part');
+  topDesktop.appendChild(headingPart);
+
+  const headingOne=document.createElement('h1');
+  headingOne.textContent=ProjectDetail[1].title;
+  headingPart.appendChild(headingOne);
+
+  
+  const headingTwo=document.createElement('h2');
+  headingTwo.classList.add('desktopHeading')
+  headingTwo.textContent=ProjectDetail[1].popTitle;
+  headingPart.appendChild(headingTwo);
+
+  const closeIcon=document.createElement('p');
+  closeIcon.classList.add('fa-times');
+  closeIcon.classList.add('para');
+  // closeIcon.src=ProjectDetail[1].closeIconImage;
+  closeIcon.textContent="X";
+  headingPart.appendChild(closeIcon);
+  const closeInconActivity="document.getElementById('card-works').style.display=block";
+  closeIcon.onclick=closeInconActivity;
+
+  const technologiesSection=document.createElement('div');
+  technologiesSection.classList.add('technologies');
+  topDesktop.appendChild(technologiesSection);
+
+  const techList=document.createElement('ul');
+  technologiesSection.appendChild(techList);
+  for(let j=0;j<ProjectDetail[1].technologies.length; j+=1)
+  {
+      const techLi=document.createElement('li');
+      techList.appendChild(techLi);
+      techLi.classList.add('tec-list');
+      const techbtn=document.createElement('button');
+      techLi.appendChild(techbtn);
+      const classes= ['button'];
```

Comment:
- [ ] it would be better if you stick to the design and use the same border color as in the Figma design 

- [ ] Figma
![image](https://user-images.githubusercontent.com/61790587/168918243-778c9783-f083-428e-bd34-e48ac466b085.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/168918287-382bde76-9c7c-41d0-89c9-2fccdbb24ec9.png)